deploy/examples/crds.yaml

```diff
@@ -5982,7 +4905,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.11.3
-  creationTimestamp: null
+  creationTimestamp: {}
```

Comment:
I'm seeing `null` converted to `{}` here. I guess this is probably `yq`, but I'm not sure how to make it stop this.

Reply:
It did cause an issue in the build, but I also think the field is unnecessary in CRD manifests, so I'm trying to just remove it.

Reply:
seems like ci is still not happy, I see logs in canary-tests/canary
```
2024-02-15 18:33:01.011255 D | op-mon: Released lock for mon orchestration
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x15992dd]

goroutine 672 [running]:
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile.func1()
	/home/runner/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.17.1/pkg/internal/controller/controller.go:116 +0x1e5
panic({0x21241e0?, 0x3da1d80?})
	/opt/hostedtoolcache/go/1.21.7/x64/src/runtime/panic.go:914 +0x21f
github.com/rook/rook/pkg/apis/ceph.rook.io/v1.(*VolumeClaimTemplate).ToPVC(0xf32f77273dc4b1af?)
	/home/runner/work/rook/rook/pkg/apis/ceph.rook.io/v1/volume.go:54 +0x1d
````